src/Backend/__init__.py

```diff
@@ -6,4 +6,4 @@
 # SPDX-License-Identifier: MPL-2.0
 # This file is part of pypowsybl-grid2opbackend.
 
-__all__ = ["powsybl_backend"]
\ No newline at end of file
+__all__ = ["powsybl_backend"]
```

Comment:
Is there a typo here ? powysybl instead of pypowsybl or is this intentional ?

Reply:
This is intentional, in fact the class is named PowsyblBackend in [powsybl_backend.py](https://github.com/powsybl/pypowsybl-grid2opbackend/blob/feature_apply_action_CICD_chronics_creator/src/Backend/powsybl_backend.py) file